docs/guide/customization.rst

```diff
@@ -43,10 +43,32 @@ in a platform specific directory, which you can see the path for by running::
     log_file_format = %(asctime)s, %(origin)s, %(levelname)s, %(message)s
   <BLANKLINE>
 
-To maintain your own customizations place a copy of the default sunpyrc file
-into the *first* path printed above.
+Do not edit the default file (the first in the "FILES USED:" list above) directly as every time you install or update sunpy, this file will be overwritten.
+
+Depending on your system, it may be useful to have a site-wide configuration file.
+If it is used, it will be on the "FILES USED:" list below the default file.
+To find your system's site configuration path for sunpy, use:
+
+.. code-block:: python
+
+    >>> from sunpy.extern.appdirs import AppDirs
+    >>> AppDirs('sunpy', 'sunpy').site_config_dir  # doctest: +SKIP
+
+In Unix, the site and user configuration paths follow the `XDG specifications <https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html>`__.
```

Comment:
My only comment here would be that it might be better to talk about the user config first, as I think that's going to apply to more people?

Reply:
My idea was to have them in the order they are read into the configuration, but I agree, for most users the site settings will probably not be important. Changed the order in e9726bc43.